pyproject.toml

```diff
@@ -117,7 +119,7 @@ django-stubs = "^1.9.0"
 djangorestframework-stubs = "^1.4.0"
 
 [tool.poetry.extras]
-flp = ["reporters-db", "juriscraper"]
+flp = ["reporters-db", "juriscraper", "judge-pics"]
```

Comment:
I'm guessing you put this here manually, right, and that you added the climage and judge-pics manually too? If so, you should revert changes to this file and the lock file, and do a `poetry add judge-pics`. That'll take care of the rest.

You can delete the entire `[tool.poetry.extras]` line too. I'm almost certain we're not using that.